packages/sdk-cli/src/commands/connect.ts

```diff
@@ -11,7 +11,18 @@ export const connectAction = async (
   deviceType: DeviceType,
   hostConnections: HostConnections,
 ) => {
-  const hosts = await developerRelay.hosts();
+  let hosts: {
+    appHost: developerRelay.Host[];
+    companionHost: developerRelay.Host[];
+  };
+
+  try {
+    hosts = await developerRelay.hosts();
+  } catch (error) {
+    cli.log(`Could not find any ${deviceType} hosts: ${error.message}`);
```

Comment:
This error message is misleading. It reads like the list was empty, which is not correct.
```suggestion
    cli.log(`An error was encountered when loading the list of available ${deviceType} hosts: ${error.message}`);
```